<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Radius:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="roundness">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="outline">
       <property name="text">
        <string>Outline around window</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Outline strength:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="alpha">
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="darkTheme">
       <property name="text">
        <string>Use black border (dark theme mode)</string>
       </property>
      </widget>
     </item> 
     <item>
      <widget class="QCheckBox" name="disabledForMaximized">
       <property name="text">
        <string>Disable for maximized windows</string>
       </property>
      </widget>
     </item> 
     <item>
      <widget class="QComboBox" name="cornersType">
      </widget>
     </item> 
     <item>
      <widget class="QLabel" name="squircleRatioLabel">
       <property name="text">
        <string>Squircle ratio</string>
       </property>
      </widget>
     </item> 
     <item>
      <widget class="QSlider" name="squircleRatio">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="pageStep">
        <number>12</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>12</number>
       </property>
      </widget>
     </item>    
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
